server/src/main/java/com/linecorp/centraldogma/server/internal/api/HttpApiUtil.java

```diff
@@ -153,6 +158,18 @@ private static HttpResponse newResponse0(HttpStatus status,
 
         node.put("message", message != null ? message : status.reasonPhrase());
 
+        // TODO(hyangtack) Need to introduce a new field such as 'stackTrace' in order to return
+        //                 the stack trace of the cause to the trusted client.
+        if (status == HttpStatus.INTERNAL_SERVER_ERROR) {
+            if (cause != null) {
+                logger.warn("Returning an internal server error with a cause:", cause);
+            } else if (message != null) {
+                logger.warn("Returning an internal server error with a message: {}", message);
```

Comment:
- What if `message != null && cause != null`?
- Could we also add `RequestContext` in the beginning of the log messages?